docs/user-guide/time-and-costs/progress-tracking/README.md

```diff
@@ -2,93 +2,111 @@
-Instead, the **% Complete** value will be set automatically based on the work package
-status configuration page.
+This allows you to edit %&nbsp;Complete (by changing status) or Work and get a preview of the updated Remaining work before saving changes.
 
-Within the Administration, navigate to Work package -> Status and select the status you want to edit regarding the progress tracking. Here you can select the desired progress percentage from the drop-down menu in the progress line and add it to the status. Please do not forget to save your changes.
+>**Note:** In the upcoming version, statuses cannot have an empty %&nbsp;Complete values in status-based progress reporting mode.When upgrading, all statuses that do not have a value will take the default value of 0%.
+
+
+## Hierarchy totals
+
+OpenProject will automatically show totals for Work, Remaining work and % Complete in a work package hierarchy (any parent with children). These appears in a work package table as a number with a Σ sign next to it, indicating that it is a sum of the values of the parent _and_ children.
+
+**[IMG: Hierarchy example]**
+
+> **Note**: The total %&nbsp;Complete value of a hierarchy is a weighted average tied to Work. For example, a feature with Work set to 50h that is 30% done will advance the sum of %&nbsp;Complete of the parent more than a feature with Work set to 5h that is 70% done. 
+
+
+### Excluding certain work packages from totals
+
+In some cases, you might want to exclude certain work packages (like those with status *rejected*) from sum calculations of the parent. Administrators can define these exclusions by going to the [Administration settings for any status] and check a new option called "Exclude from calculation of totals in hierarchy". All work packages with this status will then be excluded when calculating the sum value for the parent (for all fields: Work, Remaining work and %&nbsp;Complete).
+
+A small info icon will appear next to excluded values to remind you of this fact:
+
+**[IMG: Excluded work packages warning]**
+
+## Changing modes 
+
+When an administrator changes modes from Status-based or Work-based or vice-versa, the % Complete values might be in a "transitional stage". 
+
+## Status- to work-based
+
+In Status-based mode, it is possible for work packages to have a %&nbsp;Complete value (defined by the status) without having values for Work or Remaining work. In other words, the Work and Remaining work can be empty. 
+
+However, when switching to work-based mode, work packages with a %&nbsp;Complete value also need to have values for Work and Remaining work (since the %&nbsp;Complete is not directly editable but is rather derived from the other two values). 
+
+In this case, OpenProject will retain the original value for %&nbsp;Complete that was there mode change, but will not allow you to edit it. Instead, you will be able to input Work and Remaining work, thereby overwriting the previous value with a new computed value:
+
```

Comment:
```suggestion
When switching to Work-based mode, OpenProject will retain the value for %&nbsp;Complete that was set with status. It can then be modified by inputing Work and Remaining work, thereby overwriting the previous value with a new computed value:
```

I found the initial text too complicated to read.

If you do not retain this text, then there are some missing words in "that was there mode change". It does not sound correct to me.

Reply:
Retained! Thanks.